app/forms/idv/api_image_upload_form.rb

```diff
@@ -0,0 +1,78 @@
+module Idv
+  class ApiImageUploadForm
+    include ActiveModel::Model
+    include ActionView::Helpers::TranslationHelper
+
+    IMAGE_KEYS = %i[
+      front
+      back
+      selfie
+    ].freeze
+
+    validates_presence_of :front
+    validates_presence_of :back
+    validates_presence_of :selfie, if: :liveness_checking_enabled?
```

Comment:
I used `validate_prescence_of` because it's built-in behavior that is generally good.

However, the default error message is not so good, it ends up looking like:

> Front Please fill in this field.

We should be able to define a model name and make a custom error message for these so I'm not super worried